src/QueryProvider.tsx

```diff
@@ -25,8 +25,13 @@ export const QueryProvider = ({ children }: QueryProviderProps) => {
       queryCache: new QueryCache({
         onError: (error: any, query) => {
           const queryErrorMessage = query.meta?.errorMessage;
+          const handleError = query.meta?.handleError;
```

Comment:
Skulle hatt type for disse, men for å få til det må vi bruke type aughemtation, som krever bump til v5 først: https://tanstack.com/query/v5/docs/framework/react/typescript#registering-global-meta
Ikke noe i veien for å bumpe da, så det burde vi se på snart™️

Reply:
Er ikke typen på feilen AxiosError?

Reply:
jo, men fikk feil når jeg satte `AxiosError<any>` der (altså på linje 27) 🤔